Octokit.Tests/Reactive/ObservableRepositoryPagesClientTests.cs

```diff
@@ -160,6 +164,47 @@ public void EnsuresNonNullArguments()
 
                 Assert.Throws<ArgumentNullException>(() => client.GetLatest(null, "name"));
                 Assert.Throws<ArgumentNullException>(() => client.GetLatest("owner", null));
+
+                Assert.Throws<ArgumentException>(() => client.GetLatest("", "name"));
+                Assert.Throws<ArgumentException>(() => client.GetLatest("owner", ""));
+            }
+        }
+
+        public class TheRequestPageBuildMethod
+        {
+            [Fact]
+            public async Task PostsToCorrectUrl()
+            {
+                var gitHubClient = Substitute.For<IGitHubClient>();
+                var client = new ObservableRepositoryPagesClient(gitHubClient);
+
+                client.RequestPageBuild("fake", "repo");
```

Comment:
We don't seem to be `await`-ing in these tests. Let's add those in or drop the outer `async`...